Added a index.html file and added text in readme.md

prasannanjalikummari opened the pull request:
Hey Nigel, added a new file and added text in readme file.

prasannanjalikummari committed:
Update README.md

Added text in line 3, 4

prasannanjalikummari committed:
Merge pull request #1 from prasannanjalikummari/prasannanjalikummari-patch-1

Update README.md

prasannanjalikummari committed:
Create index.html

Added a new html

nigelquinlan reviewed (commented):
Hi Anjali,
Please add some more text to the index.html file.
Then I can approve.
Cheers, Nigel

prasannanjalikummari committed:
Update README.md

Added more text in line 6

prasannanjalikummari committed:
Update index.html

Added html content

prasannanjalikummari committed:
This commit adds html content in the file

prasannanjalikummari committed:
This commit adds a file with steps to fork other repos

prasannanjalikummari commented:
Hi Nigel,  

I am practicing forking, and pull request as part of homework via terminal and GitHub. So I added test content file with the steps.Also, I have updated index.html as per your requests
